classes/search/cmsfield.php

```diff
@@ -28,7 +31,7 @@
  * @copyright  2024 Catalyst IT
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-class cmsfield extends \core_search\base_mod {
+class cmsfield extends \core_search\base_activity {
```

Comment:
can you please rename `cmsfield` to `activity` to be consistent with other activities?

Reply:
Renamed.